server/prisma/datamodel.prisma

```diff
@@ -3,9 +3,31 @@ type User {
   email: String! @unique
   password: String!
   name: String!
+  campaigns: [Campaign!]!
 }
 
-type Post {
+type Event{
+   id: ID! @id
+   title: String! @unique
+   createdAt: DateTime! @createdAt
+   location: String!
+   image: String!
+   description: String!
+   comments: [Comment!]!
```

Comment:
Does your user need to related to the events?

Reply:
No, the user has no relation with the events. But, has direct relation with the campaign. 
Whether the user is signed in or not, the events section on the Campaign appears the same way.

Reply:
ok thanks.